hibike/flash.sh

```diff
@@ -1,10 +1,57 @@
-#/bin/bash
+#!/bin/bash
 
+NAME="$0"
+SENSOR_TYPES="$(ls devices/)"
```

Comment:
there's a script to list sensor types, actually: /utils/device_names.sh.

It's true that it does the same thing as line 4.  But it's the script that ubuntu call when it tries to tab-complete.

probably should use the same script.